README.md

```diff
@@ -23,46 +23,64 @@ The API reference is located at [realm.io/docs/javascript/latest/api/](https://r
+* **Have a bug to report?** [Open an issue](https://github.com/realm/realm-js/issues/new). If possible, include the version of Realm, a full log, the Realm file, and a project that shows the issue.
+* **Have a feature request?** [Open an issue](https://github.com/realm/realm-js/issues/new). Tell us what the feature should do, and why you want the feature.
+* Sign up for our [**Community Newsletter**](https://go.pardot.com/l/210132/2017-04-26/3j74l) to get regular tips, learn about other use-cases and get alerted of blog posts and tutorials about Realm.
 
 ## Building Realm
 
 In case you don't want to use the precompiled version on npm, you can build Realm yourself from source. You’ll need an Internet connection the first time you build in order to download the core library.
 
 Prerequisites:
-- Xcode 7.2+
-- Android SDK 23+
-- [Android NDK 10e](https://developer.android.com/ndk/downloads/older_releases)
+* Xcode 7.2+
+* nodejs
+* nvm (on Mac)
+* cocoapods (on Mac)
+* Android SDK 23+
+* [Android NDK 10e](https://developer.android.com/ndk/downloads/older_releases)
 
-First clone this repository:
+Clone RealmJS repository:
 
 ```
 git clone https://github.com/realm/realm-js.git
+cd realm-js
+git submodule update --init --recursive
 ```
 
-Then in the cloned directory:
+Note: On Windows the RealmJS repo should be cloned with symlinks enabled 
+```
+#run in elevated command prompt
+git clone -c core.symlinks=true https://github.com/realm/realm-js
+```
 
+or manually create the symlinks using directory junctions if you already have the repo cloned.
 ```
-git submodule update --init --recursive
+#run in elevated command prompt
+cd realm-js\react-native\android\src\main\jni
+mklink /j "src" "../../../../../src/"
+mklink /j "vendor" "../../../../../vendor"
+cd realm-js\tests\react-test-app\android\app\src\main
+mklink /j assets "../../../../../data"
 ```
 
 Note: If you have cloned the repo previously make sure you remove your node_modules directory since it may contain stale dependencies which may cause the build to fail.
 
 ### Building for iOS:
-- Open `react-native/ios/RealmReact.xcodeproj` in Xcode
-- Select `RealmReact` under `Targets`
-- Build: `⌘ + B`
+* Open `react-native/ios/RealmReact.xcodeproj` in Xcode
+* Select `RealmReact` under `Targets`
+* Build: `⌘ + B`
 
 ### Building for Android:
-- `cd react-native/android`
-- `./gradlew publishAndroid`
-- The compiled version of the Android module is here: `<project-root>/android`
+* `cd react-native/android`
+* `./gradlew publishAndroid`
+* The compiled version of the Android module is here: `<project-root>/android`
 
 ### Building for nodejs:
 Be sure you have python2.7 as the default python. 3.x won't work, and it's not enough to use `--python=python2.7` as parameter to npm.
+```
+brew install python@2
+```
```

Comment:
For Windows should I do the `npm install --build-from-source=realm` before or after the additional steps described below?

Reply:
before